extension-manifest-v3/src/background/stats.js

```diff
@@ -253,7 +269,7 @@ function setupTabStats(tabId, request) {
     tabStats.delete(tabId);
   }
 
-  updateIcon(tabId);
+  updateIcon(tabId, true);
```

Comment:
Is the "true" parameter intentional? From the function signature, it has only argument as far as I can tell.

Reply:
Sorry, it's a left over when I was fixing the issue with icon (didn't help and was not needed). I'll clear it up.